The 'rc' variable is unused according to GCC. Remove it.

Signed-off-by: Michael-Luke Jones

Index: linux-2.6.22-rc2-git3-armeb/drivers/ata/pata_ixp4xx_cf.c
===================================================================
--- linux-2.6.22-rc2-git3-armeb.orig/drivers/ata/pata_ixp4xx_cf.c	2007-05-23 14:33:19.000000000 -0700
+++ linux-2.6.22-rc2-git3-armeb/drivers/ata/pata_ixp4xx_cf.c	2007-05-24 04:32:39.000000000 -0700
@@ -178,7 +178,6 @@
 	struct ata_host *host;
 	struct ata_port *ap;
 	struct ixp4xx_pata_data *data = pdev->dev.platform_data;
-	int rc;
 
 	cs0 = platform_get_resource(pdev, IORESOURCE_MEM, 0);
 	cs1 = platform_get_resource(pdev, IORESOURCE_MEM, 1);
